[bitnami/openldap] Add support for setting the password crypt format

mistial-dev opened the pull request:
### Description of the change

This patch adds the configuration option "LDAP_PASSWORD_CRYPT_SALT_FORMAT", allowing the crypt salt format to be set in the container environment variables.

Additionally, this patch changes the default for password encoding from SHA (which is now retired by NIST) to salted SHA-256 using crypt() (which defaults to 5,000 rounds).  This change is also applied to administrative passwords (admin, config admin, accesslog admin).

### Benefits

NIST recommends moving away from SHA-1 due to insecurity.

> The SHA-1 algorithm, one of the first widely used methods of protecting electronic information, has reached the end of its useful life, according to security experts at the National Institute of Standards and Technology (NIST). The agency is now recommending that IT professionals replace SHA-1, in the limited situations where it is still used, with newer algorithms that are more secure.  

While it is possible to use a third party module for SHA-256 and SHA-512 from an atlassian employee, it is simpler and more compatible to use {CRYPT} support.  This allows the number of rounds to be user-configured for increased attack resistance, allows the algorithm to be specified, and provides the hash in a way that other applications using compatible crypt() implementations can interoperate with.

This improves the default security posture while supporting existing configurations.

### Possible drawbacks

Applications designed to work directly with hash values (for example, the SHA or MD5 hash) will not work unless they also have support for crypt passwords.  The previous behaviour can be restored by setting LDAP_PASSWORD_HASH to "SSHA".

Note that applications that bind as a user with a simple password should be unaffected by this change.  Only applications using the password hash directly may be affected.

### Additional information

To take advantage of the new security defaults, LDAP_CONFIGURE_PPOLICY and LDAP_PPOLICY_HASH_CLEARTEXT should be set to yes.  This will result in user passwords automatically being hashed, even if they are set in plaintext.  This applies to users created through the LDAP_USERS and LDAP_PASSWORDS environment variables.

![20351936-3250-4940-912B-B61BF8E994D3](https://github.com/user-attachments/assets/c98e7624-357e-437a-b84c-a8405587dfe5)

mistial-dev commented:
I kept the automatic plaintext conversion disabled in the interest of compatibility.  I could change that as well to make this more secure out of the box, or add a LDAP_HARDENED environment variable that sets some sensible security defaults, including adding ACLs.

Unless you need something like MSCHAPv2 support (VPN for windows, mainly), it's a very good idea to have ACLs set which prohibit reading the contents of the userPassword attribute.  If I were doing a hardened config, I'd also be tempted to limit anonymous binding and disable enumeration by users (permitting search by email or username), and adding an ou for service accounts that had full access.

andresbono commented:
Hi, thank you very much for this PR, it looks like a very nice addition! I will run some manual tests on my side and I'll get back to you.

Thanks again!

andresbono commented on bitnami/openldap/2.5/debian-12/rootfs/opt/bitnami/scripts/libopenldap.sh:
```diff
@@ -633,9 +634,7 @@ ldap_initialize() {
-        if [[ ! "$LDAP_PASSWORD_HASH" == "{SSHA}" ]]; then
-            ldap_configure_password_hash
-        fi
+        ldap_configure_password_hash
```
I guess it does not hurt to always add those configs no matter the algorithm, right?

mistial-dev replied:
Making it explicit should not hurt.

andresbono commented on bitnami/openldap/2.5/debian-12/rootfs/opt/bitnami/scripts/libopenldap.sh:
```diff
@@ -112,9 +113,9 @@ done
-export LDAP_ENCRYPTED_ADMIN_PASSWORD="$(echo -n $LDAP_ADMIN_PASSWORD | slappasswd -n -T /dev/stdin)"
-export LDAP_ENCRYPTED_CONFIG_ADMIN_PASSWORD="$(echo -n $LDAP_CONFIG_ADMIN_PASSWORD | slappasswd -n -T /dev/stdin)"
-export LDAP_ENCRYPTED_ACCESSLOG_ADMIN_PASSWORD="$(echo -n $LDAP_ACCESSLOG_ADMIN_PASSWORD | slappasswd -n -T /dev/stdin)"
+export LDAP_ENCRYPTED_ADMIN_PASSWORD="$(echo -n $LDAP_ADMIN_PASSWORD | slappasswd -c '$5$%.16s' -n -T /dev/stdin)"
+export LDAP_ENCRYPTED_CONFIG_ADMIN_PASSWORD="$(echo -n $LDAP_CONFIG_ADMIN_PASSWORD | slappasswd -c '$5$%.16s' -n -T /dev/stdin)"
+export LDAP_ENCRYPTED_ACCESSLOG_ADMIN_PASSWORD="$(echo -n $LDAP_ACCESSLOG_ADMIN_PASSWORD | slappasswd -c '$5$%.16s' -n -T /dev/stdin)"
```
If `LDAP_PASSWORD_CRYPT_SALT_FORMAT` is configurable, should it be used here instead of hardcoding the value?

mistial-dev replied:
This approach forced the use of the higher security default for the admin password.  I could make it configurable, but it wouldn't force the higher security option for the admin password.

andresbono replied:
I see. I think it should be configurable. If users may need different formats for admin and the rest, maybe we can define a new env-var: `LDAP_ADMIN_PASSWORD_CRYPT_SALT_FORMAT`. What do you think?

mistial-dev replied:
That makes sense.

mistial-dev replied:
> I see. I think it should be configurable. If users may need different formats for admin and the rest, maybe we can define a new env-var: `LDAP_ADMIN_PASSWORD_CRYPT_SALT_FORMAT`. What do you think?

Added.  Additionally, I changed the defaults to enable ppolicy and hash plaintext passwords by default.

It can be turned off easily enough, either through environment variables or ldapmodify after the fact.  Sane defaults are better, and with Microsoft Active Directory using hashed passwords, there's precedent for not doing plaintext passwords.  It also works fine for shadow support, and the use of CRYPT instead of something weird increases compatibility when used elsewhere.

andresbono reviewed (commented):
Could you please check my comments? Thanks!

mistial-dev commented:
> bitnami/openldap/2.5/debian-12/rootfs/opt/bitnami/scripts/libopenldap.sh

Looking at them now.

andresbono reviewed (commented):
Thank you, could you please check my comment?

mistial-dev commented on bitnami/openldap/2.6/debian-12/rootfs/opt/bitnami/scripts/libopenldap.sh:
```diff
@@ -76,10 +76,12 @@ export LDAP_REQUIRE_TLS="${LDAP_REQUIRE_TLS:-no}"
+export LDAP_CONFIGURE_PPOLICY="${LDAP_CONFIGURE_PPOLICY:-yes}"
 export LDAP_PPOLICY_USE_LOCKOUT="${LDAP_PPOLICY_USE_LOCKOUT:-no}"
-export LDAP_PPOLICY_HASH_CLEARTEXT="${LDAP_PPOLICY_HASH_CLEARTEXT:-no}"
+export LDAP_PPOLICY_HASH_CLEARTEXT="${LDAP_PPOLICY_HASH_CLEARTEXT:-yes}"
```
Applying this through the policy means that it's applied when necessary.  It doesn't force a particular hashed format if non-plaintext passwords are applied.

mistial-dev commented on bitnami/openldap/2.6/debian-12/rootfs/opt/bitnami/scripts/libopenldap.sh:
```diff
@@ -76,10 +76,12 @@ export LDAP_REQUIRE_TLS="${LDAP_REQUIRE_TLS:-no}"
+export LDAP_PASSWORD_HASH="${LDAP_PASSWORD_HASH:-{CRYPT\}}"
+export LDAP_PASSWORD_CRYPT_SALT_FORMAT="${LDAP_PASSWORD_CRYPT_SALT_FORMAT:-\$5\$%.16s}"
+export LDAP_ADMIN_PASSWORD_CRYPT_SALT_FORMAT="${LDAP_ADMIN_PASSWORD_CRYPT_SALT_FORMAT:-\$5\$%.16s}"
+export LDAP_CONFIGURE_PPOLICY="${LDAP_CONFIGURE_PPOLICY:-yes}"
```
Setting this as the default ensures that password hashing is used even when the environment variables are used to pass credentials and no policy is set.  Leaving plaintext passwords around is risky.

andresbono commented:
@mistial-dev, just one minor thing, The DCO check failed. Please read https://github.com/bitnami/containers/pull/70398/checks?check_run_id=30415074724 to see how you can fix that. Thanks!

Commit:
[bitnami/postgresql] Release postgresql-13.16.0-debian-12-r13 (#72766)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/postgresql-repmgr] Release postgresql-repmgr-15.8.0-debian-12-r15 (#72768)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/postgresql-repmgr] Release postgresql-repmgr-14.13.0-debian-12-r18 (#72767)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/postgresql] Release postgresql-12.20.0-debian-12-r14 (#72769)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/nessie] Release nessie-0.99.0-debian-12-r0 (#72770)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/postgresql-repmgr] Release postgresql-repmgr-12.20.0-debian-12-r17 (#72771)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/postgresql] Release postgresql-14.13.0-debian-12-r12 (#72772)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/nessie-utils] Release nessie-utils-0.99.0-debian-12-r0 (#72773)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/postgresql-repmgr] Release postgresql-repmgr-14.13.0-debian-12-r19 (#72774)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/postgresql] Release postgresql-15.8.0-debian-12-r13 (#72775)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/postgresql-repmgr] Release postgresql-repmgr-16.4.0-debian-12-r18 (#72776)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/postgresql] Release postgresql-13.16.0-debian-12-r14 (#72777)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/postgresql-repmgr] Release postgresql-repmgr-15.8.0-debian-12-r16 (#72778)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/fluxcd-source-controller] Release fluxcd-source-controller-1.4.1-debian-12-r0 (#72779)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/postgresql-repmgr] Release postgresql-repmgr-13.16.0-debian-12-r14 (#72780)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/opensearch] Release opensearch-2.17.0-debian-12-r2 (#72781)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/aws-cli] Release aws-cli-2.17.60-debian-12-r0 (#72782)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/fluxcd-image-reflector-controller] Release fluxcd-image-reflector-controller-0.33.0-debian-12-r0 (#72783)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/postgresql] Release postgresql-16.4.0-debian-12-r12 (#72784)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/fluxcd-helm-controller] Release fluxcd-helm-controller-1.1.0-debian-12-r0 (#72785)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/postgresql-repmgr] Release postgresql-repmgr-16.4.0-debian-12-r19 (#72786)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/cassandra] Release cassandra-4.1.7-debian-12-r0 (#72787)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/cassandra] Release cassandra-4.0.14-debian-12-r0 (#72788)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/clickhouse] Release clickhouse-24.9.1-debian-12-r0 (#72789)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/fluxcd-image-automation-controller] Release fluxcd-image-automation-controller-0.39.0-debian-12-r0 (#72790)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/cilium] Release cilium-1.16.2-debian-12-r1 (#72791)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/fluent-bit] Release fluent-bit-3.1.8-debian-12-r1 (#72792)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/couchdb] Release couchdb-3.4.1-debian-12-r0 (#72793)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/postgresql-repmgr] Release postgresql-repmgr-16.4.0-debian-12-r20 (#72795)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/fluxcd-kustomize-controller] Release fluxcd-kustomize-controller-1.4.0-debian-12-r0 (#72797)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/ghost] Release ghost-5.95.0-debian-12-r0 (#72798)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/gotrue] Release gotrue-2.162.0-debian-12-r0 (#72799)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/fluent-bit] Release fluent-bit-3.1.9-debian-12-r0 (#72800)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/python] Release python-3.12.6-debian-12-r8 (#72801)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/python] Release python-3.11.10-debian-12-r7 (#72802)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/python] Release python-3.9.20-debian-12-r5 (#72803)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/python] Release python-3.8.20-debian-12-r6 (#72804)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/python] Release python-3.10.15-debian-12-r5 (#72805)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/kuberay-apiserver] Release kuberay-apiserver-1.2.2-debian-12-r0 (#72808)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/kuberay-operator] Release kuberay-operator-1.2.2-debian-12-r0 (#72809)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/scylladb] Release scylladb-6.0.4-debian-12-r0 (#72810)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/cypress] Release cypress-13.15.0-debian-12-r1 (#72812)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/postgresql] Release postgresql-16.4.0-debian-12-r13 (#72814)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

dependabot[bot] committed:
Bump actions/checkout from 4.1.7 to 4.2.0 (#72813)

Bumps [actions/checkout](https://github.com/actions/checkout) from 4.1.7 to 4.2.0.
- [Release notes](https://github.com/actions/checkout/releases)
- [Changelog](https://github.com/actions/checkout/blob/main/CHANGELOG.md)
- [Commits](https://github.com/actions/checkout/compare/692973e3d937129bcbf40652eb9f2f61becf3332...d632683dd7b4114ad314bca15554477dd762a938)

---
updated-dependencies:
- dependency-name: actions/checkout
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/fluxcd-notification-controller] Release fluxcd-notification-controller-1.4.0-debian-12-r0 (#72815)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/chainloop-control-plane-migrations] Release chainloop-control-plane-migrations-0.96.13-debian-12-r0 (#72816)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/chainloop-artifact-cas] Release chainloop-artifact-cas-0.96.13-debian-12-r0 (#72817)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/chainloop-control-plane] Release chainloop-control-plane-0.96.13-debian-12-r0 (#72818)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/php-fpm] Release php-fpm-8.1.30-debian-12-r0 (#72819)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/scylladb] Release scylladb-6.1.2-debian-12-r1 (#72820)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/scylladb] Release scylladb-6.1.2-debian-12-r2 (#72821)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/tensorflow-serving] Release tensorflow-serving-2.17.0-debian-12-r7 (#72822)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/cassandra] Release cassandra-5.0.1-debian-12-r0 (#72823)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/clickhouse] Release clickhouse-24.3.11-debian-12-r1 (#72824)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/clickhouse] Release clickhouse-24.9.1-debian-12-r1 (#72825)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/mastodon] Release mastodon-4.2.13-debian-12-r0 (#72826)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/sonarqube] Release sonarqube-9.9.7-debian-12-r0 (#72827)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/kubescape] Release kubescape-3.0.18-debian-12-r0 (#72828)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/sonarqube] Release sonarqube-10.7.0-debian-12-r0 (#72829)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/seaweedfs] Release seaweedfs-3.74.0-debian-12-r0 (#72830)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/wordpress-nginx] Release wordpress-nginx-6.6.2-debian-12-r6 (#72831)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/wordpress] Release wordpress-6.6.2-debian-12-r6 (#72832)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/aws-cli] Release aws-cli-1.34.30-debian-12-r0 (#72833)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/nessie] Release nessie-0.99.0-debian-12-r1 (#72834)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/jupyterhub] Release jupyterhub-5.2.0-debian-12-r0 (#72835)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/nessie] Release nessie-0.99.0-debian-12-r2 (#72836)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/jupyter-base-notebook] Release jupyter-base-notebook-5.2.0-debian-12-r0 (#72837)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/python] Release python-3.12.7-debian-12-r0 (#72838)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/postgresql] Release postgresql-16.4.0-debian-12-r14 (#72840)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/grafana] Release grafana-10.4.10-debian-12-r0 (#72841)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/acmesolver] Release acmesolver-1.15.3-debian-12-r4 (#72842)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/alertmanager] Release alertmanager-0.27.0-debian-12-r23 (#72843)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/airflow-exporter] Release airflow-exporter-0.20220314.0-debian-12-r48 (#72844)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/apache-exporter] Release apache-exporter-1.0.9-debian-12-r1 (#72845)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/apisix-dashboard] Release apisix-dashboard-3.0.1-debian-12-r46 (#72846)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/apisix-ingress-controller] Release apisix-ingress-controller-1.8.2-debian-12-r17 (#72847)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/argo-cd] Release argo-cd-2.12.4-debian-12-r1 (#72848)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/argo-workflow-controller] Release argo-workflow-controller-3.5.11-debian-12-r1 (#72849)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/argo-workflow-exec] Release argo-workflow-exec-3.5.11-debian-12-r1 (#72850)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/blackbox-exporter] Release blackbox-exporter-0.25.0-debian-12-r17 (#72851)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/cainjector] Release cainjector-1.15.3-debian-12-r3 (#72852)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/cert-manager-webhook] Release cert-manager-webhook-1.15.3-debian-12-r3 (#72853)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/cert-manager] Release cert-manager-1.15.3-debian-12-r4 (#72854)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/chainloop-artifact-cas] Release chainloop-artifact-cas-0.96.13-debian-12-r1 (#72855)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/chainloop-control-plane] Release chainloop-control-plane-0.96.13-debian-12-r1 (#72856)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/charts-syncer] Release charts-syncer-2.0.2-debian-12-r1 (#72857)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/cilium-operator] Release cilium-operator-1.16.2-debian-12-r1 (#72858)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/cilium-proxy] Release cilium-proxy-1.29.9-debian-12-r1 (#72859)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/cilium] Release cilium-1.16.2-debian-12-r2 (#72860)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/cluster-autoscaler] Release cluster-autoscaler-1.31.0-debian-12-r4 (#72861)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/configmap-reload] Release configmap-reload-0.14.0-debian-12-r1 (#72862)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/consul-exporter] Release consul-exporter-0.12.1-debian-12-r6 (#72863)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/consul] Release consul-1.19.2-debian-12-r4 (#72864)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/contour] Release contour-1.27.4-debian-12-r11 (#72865)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/contour] Release contour-1.28.6-debian-12-r5 (#72866)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/contour] Release contour-1.29.2-debian-12-r5 (#72867)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/contour] Release contour-1.30.0-debian-12-r6 (#72868)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/cosign] Release cosign-2.4.0-debian-12-r5 (#72869)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/dex] Release dex-2.41.1-debian-12-r5 (#72870)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/elasticsearch-exporter] Release elasticsearch-exporter-1.8.0-debian-12-r1 (#72871)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/etcd] Release etcd-3.4.34-debian-12-r2 (#72872)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/etcd] Release etcd-3.5.16-debian-12-r2 (#72873)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/external-dns] Release external-dns-0.15.0-debian-12-r2 (#72874)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/fluxcd-source-controller] Release fluxcd-source-controller-1.4.1-debian-12-r1 (#72875)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/fluxcd-helm-controller] Release fluxcd-helm-controller-1.1.0-debian-12-r1 (#72876)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/fluxcd-image-automation-controller] Release fluxcd-image-automation-controller-0.39.0-debian-12-r1 (#72877)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/fluxcd-image-reflector-controller] Release fluxcd-image-reflector-controller-0.33.0-debian-12-r1 (#72878)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/fluxcd-kustomize-controller] Release fluxcd-kustomize-controller-1.4.0-debian-12-r1 (#72879)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/fluxcd-notification-controller] Release fluxcd-notification-controller-1.4.0-debian-12-r1 (#72880)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/gitea] Release gitea-1.22.2-debian-12-r3 (#72881)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/gitlab-runner-helper] Release gitlab-runner-helper-17.4.0-debian-12-r1 (#72882)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/gitlab-runner] Release gitlab-runner-17.4.0-debian-12-r1 (#72883)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/golang] Release golang-1.23.2-debian-12-r0 (#72884)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/google-cloud-sdk] Release google-cloud-sdk-0.495.0-debian-12-r0 (#72885)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/gotrue] Release gotrue-2.162.0-debian-12-r1 (#72886)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/grafana-loki] Release grafana-loki-3.2.0-debian-12-r1 (#72887)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/grafana-mimir] Release grafana-mimir-2.13.0-debian-12-r10 (#72888)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/grafana-operator] Release grafana-operator-5.13.0-debian-12-r1 (#72889)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/grafana-tempo-query] Release grafana-tempo-query-2.6.0-debian-12-r3 (#72890)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/grafana-tempo-vulture] Release grafana-tempo-vulture-2.6.0-debian-12-r3 (#72891)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/grafana-tempo] Release grafana-tempo-2.6.0-debian-12-r3 (#72892)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/grafana] Release grafana-10.4.10-debian-12-r1 (#72893)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/grafana] Release grafana-11.2.2-debian-12-r0 (#72894)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/haproxy] Release haproxy-2.9.11-debian-12-r2 (#72895)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/haproxy] Release haproxy-3.0.5-debian-12-r2 (#72896)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/harbor-adapter-trivy] Release harbor-adapter-trivy-2.11.1-debian-12-r3 (#72897)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/harbor-core] Release harbor-core-2.11.1-debian-12-r6 (#72898)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/golang] Release golang-1.22.8-debian-12-r0 (#72899)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/harbor-exporter] Release harbor-exporter-2.11.1-debian-12-r6 (#72900)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/harbor-jobservice] Release harbor-jobservice-2.11.1-debian-12-r5 (#72901)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/harbor-registry] Release harbor-registry-2.11.1-debian-12-r6 (#72902)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/harbor-registryctl] Release harbor-registryctl-2.11.1-debian-12-r5 (#72903)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/hubble-relay] Release hubble-relay-1.16.2-debian-12-r1 (#72904)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/hubble-ui-backend] Release hubble-ui-backend-0.13.1-debian-12-r12 (#72905)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/hyperledger-fabric-ca] Release hyperledger-fabric-ca-1.5.13-debian-12-r1 (#72906)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/hyperledger-fabric-orderer] Release hyperledger-fabric-orderer-2.5.10-debian-12-r1 (#72907)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/hyperledger-fabric-orderer] Release hyperledger-fabric-orderer-3.0.0-debian-12-r1 (#72908)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/hyperledger-fabric-peer] Release hyperledger-fabric-peer-2.5.10-debian-12-r1 (#72909)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/hyperledger-fabric-peer] Release hyperledger-fabric-peer-3.0.0-debian-12-r1 (#72910)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/hyperledger-fabric-tools] Release hyperledger-fabric-tools-2.5.10-debian-12-r1 (#72911)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/hyperledger-fabric-tools] Release hyperledger-fabric-tools-3.0.0-debian-12-r1 (#72912)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/influxdb] Release influxdb-2.7.10-debian-12-r5 (#72913)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/jaeger] Release jaeger-1.61.0-debian-12-r1 (#72914)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/kaniko] Release kaniko-1.23.2-debian-12-r7 (#72915)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/kiam] Release kiam-4.2.0-debian-12-r41 (#72916)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/kong-ingress-controller] Release kong-ingress-controller-2.12.6-debian-12-r4 (#72917)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/kong-ingress-controller] Release kong-ingress-controller-3.3.1-debian-12-r4 (#72918)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/kube-rbac-proxy] Release kube-rbac-proxy-0.18.1-debian-12-r2 (#72919)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/kube-state-metrics] Release kube-state-metrics-2.13.0-debian-12-r7 (#72920)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/kubeapps-apis] Release kubeapps-apis-2.11.0-debian-12-r7 (#72921)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/kubeapps-apprepository-controller] Release kubeapps-apprepository-controller-2.11.0-debian-12-r5 (#72922)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/kubeapps-asset-syncer] Release kubeapps-asset-syncer-2.11.0-debian-12-r5 (#72923)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/kubectl] Release kubectl-1.28.14-debian-12-r3 (#72924)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/kubectl] Release kubectl-1.29.9-debian-12-r3 (#72925)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/kubectl] Release kubectl-1.30.5-debian-12-r3 (#72926)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/kubectl] Release kubectl-1.31.1-debian-12-r3 (#72927)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/kuberay-apiserver] Release kuberay-apiserver-1.2.2-debian-12-r1 (#72928)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/kuberay-operator] Release kuberay-operator-1.2.2-debian-12-r1 (#72929)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/kubernetes-event-exporter] Release kubernetes-event-exporter-1.7.0-debian-12-r15 (#72930)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/kubescape] Release kubescape-3.0.3-debian-12-r20 (#72931)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/kubescape] Release kubescape-3.0.18-debian-12-r1 (#72932)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/memcached-exporter] Release memcached-exporter-0.14.4-debian-12-r9 (#72933)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/metallb-controller] Release metallb-controller-0.14.8-debian-12-r7 (#72934)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/metrics-server] Release metrics-server-0.7.2-debian-12-r4 (#72935)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/milvus] Release milvus-2.4.12-debian-12-r1 (#72936)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/minio-client] Release minio-client-2024.9.16-debian-12-r1 (#72937)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/minio] Release minio-2024.9.22-debian-12-r1 (#72938)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/mongodb-exporter] Release mongodb-exporter-0.41.1-debian-12-r1 (#72939)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/multus-cni] Release multus-cni-4.1.1-debian-12-r1 (#72941)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/mysqld-exporter] Release mysqld-exporter-0.15.1-debian-12-r35 (#72940)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/nats-exporter] Release nats-exporter-0.15.0-debian-12-r15 (#72942)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/nats] Release nats-2.10.21-debian-12-r1 (#72943)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/natscli] Release natscli-0.1.5-debian-12-r6 (#72944)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/nginx-exporter] Release nginx-exporter-1.3.0-debian-12-r7 (#72945)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/nginx-ingress-controller] Release nginx-ingress-controller-1.11.2-debian-12-r5 (#72946)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/node-exporter] Release node-exporter-1.8.2-debian-12-r8 (#72947)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/notation] Release notation-1.2.0-debian-12-r3 (#72948)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/oauth2-proxy] Release oauth2-proxy-7.6.0-debian-12-r23 (#72949)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/oras] Release oras-1.2.0-debian-12-r8 (#72950)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/pinniped-cli] Release pinniped-cli-0.33.0-debian-12-r4 (#72951)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/pinniped] Release pinniped-0.33.0-debian-12-r2 (#72952)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/prometheus-operator] Release prometheus-operator-0.77.1-debian-12-r1 (#72953)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/prometheus] Release prometheus-2.54.1-debian-12-r5 (#72954)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/postgres-exporter] Release postgres-exporter-0.15.0-debian-12-r44 (#72955)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/promtail] Release promtail-3.2.0-debian-12-r1 (#72956)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/pushgateway] Release pushgateway-1.10.0-debian-12-r1 (#72958)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/rabbitmq-cluster-operator] Release rabbitmq-cluster-operator-2.10.0-debian-12-r2 (#72959)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/rclone] Release rclone-1.68.1-debian-12-r1 (#72960)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/redis-exporter] Release redis-exporter-1.63.0-debian-12-r1 (#72961)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/rmq-default-credential-updater] Release rmq-default-credential-updater-1.0.4-debian-12-r29 (#72962)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/rmq-messaging-topology-operator] Release rmq-messaging-topology-operator-1.14.2-debian-12-r7 (#72963)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/sealed-secrets-kubeseal] Release sealed-secrets-kubeseal-0.27.1-debian-12-r5 (#72964)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/sealed-secrets-controller] Release sealed-secrets-controller-0.27.1-debian-12-r5 (#72965)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/seaweedfs] Release seaweedfs-3.74.0-debian-12-r1 (#72966)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/telegraf] Release telegraf-1.32.0-debian-12-r1 (#72967)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/trivy] Release trivy-0.55.2-debian-12-r1 (#72969)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/thanos] Release thanos-0.36.1-debian-12-r3 (#72968)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/vault-csi-provider] Release vault-csi-provider-1.5.0-debian-12-r3 (#72970)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/vault-k8s] Release vault-k8s-1.4.2-debian-12-r9 (#72971)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/vault] Release vault-1.17.6-debian-12-r1 (#72972)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/volsync] Release volsync-0.10.0-debian-12-r12 (#72973)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/whereabouts] Release whereabouts-0.8.0-debian-12-r6 (#72974)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/tensorflow-serving] Release tensorflow-serving-2.17.0-debian-12-r8 (#72977)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/postgresql] Release postgresql-17.0.0-debian-12-r0 (#72978)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/opensearch] Release opensearch-2.17.0-debian-12-r3 (#72980)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/oauth2-proxy] Release oauth2-proxy-7.7.0-debian-12-r0 (#72981)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/opensearch] Release opensearch-2.17.0-debian-12-r4 (#72982)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/jenkins] Release jenkins-2.462.3-debian-12-r0 (#72983)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/php-fpm] Release php-fpm-8.2.24-debian-12-r1 (#72984)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/nginx] Release nginx-1.27.2-debian-12-r0 (#72985)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/appsmith] Release appsmith-1.43.0-debian-12-r0 (#72986)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/php-fpm] Release php-fpm-8.1.30-debian-12-r1 (#72987)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/argo-workflow-cli] Release argo-workflow-cli-3.5.11-debian-12-r1 (#72988)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/php-fpm] Release php-fpm-8.3.12-debian-12-r1 (#72989)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/neo4j] Release neo4j-5.24.0-debian-12-r1 (#72990)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/redis-sentinel] Release redis-sentinel-7.4.1-debian-12-r0 (#72991)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/redis-cluster] Release redis-cluster-7.4.1-debian-12-r0 (#72992)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/redis] Release redis-7.4.1-debian-12-r0 (#72993)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/neo4j] Release neo4j-5.24.1-debian-12-r0 (#72994)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/minio-client] Release minio-client-2024.10.2-debian-12-r0 (#72995)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/redis-cluster] Release redis-cluster-6.2.16-debian-12-r0 (#72998)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/redis] Release redis-6.2.16-debian-12-r0 (#72996)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/redis-sentinel] Release redis-sentinel-6.2.16-debian-12-r0 (#72997)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/minio] Release minio-2024.10.2-debian-12-r0 (#72999)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/ruby] Release ruby-3.2.5-debian-12-r6 (#73001)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/gotrue] Release gotrue-2.162.1-debian-12-r0 (#73002)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/trivy] Release trivy-0.56.0-debian-12-r0 (#73003)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/drupal] Release drupal-10.3.6-debian-12-r0 (#73004)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/drupal] Release drupal-11.0.5-debian-12-r0 (#73005)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/clickhouse] Release clickhouse-24.9.1-debian-12-r2 (#73006)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/clickhouse] Release clickhouse-24.9.2-debian-12-r0 (#73007)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/elasticsearch] Release elasticsearch-8.15.2-debian-12-r1 (#73008)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/elasticsearch] Release elasticsearch-7.17.24-debian-12-r1 (#73009)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/acmesolver] Release acmesolver-1.16.0-debian-12-r0 (#73010)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/cert-manager] Release cert-manager-1.16.0-debian-12-r0 (#73011)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/opensearch] Release opensearch-1.3.19-debian-12-r6 (#73012)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/elasticsearch] Release elasticsearch-8.15.2-debian-12-r2 (#73013)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Trungtin1011 committed:
[bitnami][clickhouse] Fix ClickHouse bug on custom init/start script (#72646)

[bitnami][clickhouse] Correct ClickHouse behavior when user adds custom init/start script

Signed-off-by: Tin Trung Ngo <trungtinth1011@gmail.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/cainjector] Release cainjector-1.16.0-debian-12-r0 (#73014)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/cert-manager-webhook] Release cert-manager-webhook-1.16.0-debian-12-r0 (#73015)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/trivy] Release trivy-0.56.1-debian-12-r0 (#73016)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/suitecrm] Release suitecrm-8.7.0-debian-12-r0 (#73017)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/seaweedfs] Release seaweedfs-3.75.0-debian-12-r0 (#73018)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/opensearch] Release opensearch-2.17.0-debian-12-r5 (#73019)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/cosign] Release cosign-2.4.1-debian-12-r0 (#73020)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/valkey-sentinel] Release valkey-sentinel-8.0.1-debian-12-r0 (#73021)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/valkey-cluster] Release valkey-cluster-8.0.1-debian-12-r0 (#73022)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/valkey] Release valkey-8.0.1-debian-12-r0 (#73023)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/node] Release node-20.18.0-debian-12-r0 (#73024)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

Commit:
[bitnami/tensorflow-resnet] Release tensorflow-resnet-2.17.0-debian-12-r7 (#73025)

Signed-off-by: Bitnami Bot <bitnami-bot@vmware.com>
Signed-off-by: Mistial Developer <admin@mistial.dev>

mistial-dev commented:
> @mistial-dev, just one minor thing, The DCO check failed. Please read https://github.com/bitnami/containers/pull/70398/checks?check_run_id=30415074724 to see how you can fix that. Thanks!

I tried what it suggested earlier, and it ended up making a hash of things.  I'll try again.

mistial-dev commented:
> @mistial-dev, just one minor thing, The DCO check failed. Please read https://github.com/bitnami/containers/pull/70398/checks?check_run_id=30415074724 to see how you can fix that. Thanks!

@andresbono I have followed the instructions in the document.  It is now attempting to modify 2,409 files.

gongomgra commented:
Hi @mistial-dev,

Thanks for fixing the DCO issue. Can you rebase the main branch on your fork and fix the conflicts?
